tests/torch/test_sota_checkpoints.py

```diff
@@ -538,8 +538,9 @@ def openvino_preinstall(ov_data_dir):
     if ov_data_dir:
         subprocess.run("pip install -r requirements_onnx.txt", cwd=MO_DIR, check=True, shell=True)
         subprocess.run("pip install scikit-image==0.17.2", check=True, shell=True)
-        subprocess.run("{} setup.py install".format(sys.executable), cwd=ACC_CHECK_DIR, check=True, shell=True)
-
+        # Workaround to fix packaging error
+        subprocess.run("pip uninstall packaging -y", check=True, shell=True)
+        subprocess.run(f"{sys.executable} setup.py install", cwd=ACC_CHECK_DIR, check=True, shell=True)
```

Comment:
Found root couse of problem, Updating docker image. So, removed all Workaround's, please see Job 8771.

Reply:
Great! What was the root cause and what did you update in the docker image? Should it affect NNCF dockerfile?

Reply:
The components remained unchanged, just re-builded docker, One of component - packaging (one of PyTest dependency, it's not in our requirements list) was updated since last docker build, so, old version was incompatible with one dependency of Accuracy Checker. I don't think this affect NNCF.